app/Tools/FileFormatter.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Tools;
+
+use Illuminate\Support\Facades\Storage;
+
+class FileFormatter extends ConsoleCommand
+{
+    private array $rules;
+
+    private string $file;
+
+    public function __construct() {}
+
+    public function file(string $path): static
+    {
+        $this->file = Storage::path($path);
+
+        return $this;
+    }
+
+    public function importOrder(): FileFormatter
+    {
+        $this->rules[] = 'ordered_imports';
+
+        return $this;
+    }
+
+    public function arraySyntax(): FileFormatter
+    {
+        $this->rules[] = 'array_syntax';
+
+        return $this;
+    }
+
+    public function fix(): void
```

Comment:
In case it's not obvious, i'm using `base_path('vendor/bin/php-cs-fixer')` here so that we are using Mise's vendor directory where php-cs-fixer resides. This got me tearing my hair out during dev when all the tests were passsing but no formatting was happening on the laravel code 🤦‍♂️.